internal/state/store.go

```diff
@@ -333,7 +365,8 @@ func (store dbStore) PruneStates(from int64, to int64, evidenceThresholdHeight i
 		// params, otherwise they will panic if they're retrieved directly (instead of
 		// indirectly via a LastHeightChanged pointer).
 		if keepVals[h] {
-			v, err := loadValidatorsInfo(store.db, h)
+			v, tmpTime, err := loadValidatorsInfo(store.db, h)
```

Comment:
Again, I lack some context, but do we allow variable shadowing? `err` is previously defined in the function.

Reply:
So this was here from before, I don't think we have explicit rules but given that on any error before or after we return immediately it can pass here.. It would be messier to declare the vars above just to avoid overwriting this one.